Model/Carrier/LogitrailCarrier.php

```diff
@@ -117,9 +116,10 @@ public function getForm($lang = 'fi')
             join(' ', $address->getStreet()),
             $address->getPostcode(),
             $address->getCity(),
-            $address->getCompany()
+            $address->getCompany(),
+            $address->getCountryId()
         );
-        $form = $api->getForm($lang);
+        $form = $api->getForm($lang, array('total_sum' => $this->quote->getBaseGrandTotal()));
```

Comment:
After the shipping cost related selections have been made it does include the shipping cost as well. On the first execution before a drop off or pickup point has been selected it doesn't include a shipping cost because we haven't received one from logitrail yet.